fuse/baton_rouge_so.py

```diff
@@ -71,7 +64,7 @@ def fuse_events(cprr_18, cprr_21, post):
 if __name__ == "__main__":
     cprr_18 = pd.read_csv(data_file_path("match/cprr_baton_rouge_so_2018.csv"))
     cprr_20 = pd.read_csv(data_file_path("match/cprr_baton_rouge_so_2016_2020.csv"))
-    post = prepare_post_data()
+    post = pd.read_csv(data_file_path("clean/pprr_post_2020_11_06.csv"))
```

Comment:
Here as well, if we don't filter POST by agency in every fuse script then event tables of individual agencies will include all post events, which is a lot of redundancy. You need to not only filter by agency in individual fuse script, but also do it in such a way that it's not possible for there to be 2 different agencies in the output because `fuse/all.py` script will not be able to detect that kind of error. I would change it like this:
```python
agency = cprr_20.agency[0]
post = pd.read_csv(data_file_path("clean/pprr_post_2020_11_06.csv"))
post = post.loc[post.agency == agency]
```

Reply:
Thank you.